src/scripts/destroy-db.ts

```diff
@@ -0,0 +1,20 @@
+import { MongoClient } from "mongodb";
+import "dotenv/config";
+
+let databaseUrl = process.env.MONGO_URL || "";
+
+if (process.env.HEROKU_APP_NAME) {
```

Comment:
```suggestion
if (process.env.HEROKU_APP_NAME) {
```

HEROKU_APP_NAME is not in the `.env.sample`

Reply:
Speaking of which: I think we should remove elements from the `.env.sample` to only keep what is needed on the first day of the project: it might cause students to question and confuse with Heroku and Auth0. We can then ask them to update `.env.sample` and `.env`, when necessary. 
What do you think? @Fenntasy @TheRealAstoo

Reply:
I agree

Reply:
If you take it out of the `env.sample` then we should give a hint in the `README` :)

Reply:
It turns out, as I revamp the course, that they'll be already need a Heroku setup, so better keep it 😅 
But Auth0 will still be removed